mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
```

Comment:
@sanxiyn Thank you. I would have provided it, but I have been having some debilitating internet problems.

I'm working on a fork of that with changes to update the rust buildroot feature branch to be more flexible in allowing uclibc toolchains as well as some documentation on how to configure a rust toolchain for buildroot. It doesn't exactly have a checkbox you know.

@japaric So, I would be delighted to reduce the dependency on make as much as possible, and I totally understand you frustrations around time consumption. I've been writing some buildroot packages for a while, so I wouldn't have a problem altering their setup to build using the cargo build system. There's no way they'll give up using Makefiles; buildroot builds *gigabytes* worth of software for embedded environments, and there's too much momentum there. But I can't see them objecting to a buildroot package that calls cargo; we'd just be wrapping the rust build script with make, which sounds perfectly fine to me.